### ISSUE TITLE
[BUG] Files created in home directory when downloading

### ISSUE THREAD

**Issue by masytan** [bottlesdevs/Bottles#1068]
0-byte files named "Downloading" or part of the word are created in the home directory on the first run download, as well as the first time I create a application and gaming bottle. This is the Flatpak version without home access.
My overrides are filesystems=~/Games/Bottles for Bottles, and filesystems=/var/home/[username]/.themes:ro; globally.

**Installation**
 - Method: Flatpak
 - Version 2022.2.14-trento

**To Reproduce**
1. Clean install Bottles
2. Let the wizard install whatever it does

**Expected behavior**
For the program to run as it does without touching the home dir.

**Screenshots**
![2022-02-17_08-02-1645083114](https://user-images.githubusercontent.com/89985351/154431709-f73b8878-7dd8-4b54-a8ad-3532af91e8b8.png)

**Desktop (please complete the following information):**
 - OS: Fedora Silverblue
 - Version 35
 - Display server: Wayland

**Additional context**
Sway is installed over Gnome.

**Comment by masytan** [comment]
Just in terminal? The output is empty.

**Comment by mirkobrombin** [comment]
So it is using ~/.var/app/com.usebottles.bottles/data/bottles path. Also, Bottles does not have access to your home so actually these files are not placed by it.

**Comment by masytan** [comment]
Yep, I just reset bottles, and it is using data/bottles/temp, but still creating those files in home. It is definitely happening because of Bottles. Should I be reporting this to Flatpak instead?
![2022-02-17_10-02-1645091438](https://user-images.githubusercontent.com/89985351/154450881-f3f37d27-3f8f-4d7a-9580-550ad547df03.png)
![2022-02-17_10-02-1645091733](https://user-images.githubusercontent.com/89985351/154451179-23719c58-091b-4aa8-abb7-f1f3958b27b8.png)

.

**Comment by mirkobrombin** [comment]
Should be not a Flatpak problem. It's just weird. Bottles is writing in the correct temp so the files are actually placed in the right place.

The only place where "Downloading" is written is in the gui and cli just being downloaded. So what I think (but I can't explain) is that the console output is redirected to files in your homedir (I'm probably saying a heresy because I don't think that's possible). In this case, however, if so, there is a serious problem, namely the fact that Bottles is somehow evading his own Flatpak, accessing the root of your homedir (eg /home/you/) which I do not think you gave access.

**Comment by masytan** [comment]
I went ahead and reset all Bottles and global overrides and tested again, the issue persists. Here's the output of `flatpak info --show-permissions com.usebottles.bottles` now.


```
[Context]
shared=network;ipc;
sockets=x11;wayland;pulseaudio;
devices=all;
features=devel;multiarch;

[Session Bus Policy]
org.freedesktop.Notifications=talk

[System Bus Policy]
org.freedesktop.UDisks2=talk

[Environment]
LD_LIBRARY_PATH=/app/lib:/app/lib32
PATH=/app/bin:/app/utils/bin:/usr/bin

```
I don't know what I might have done to cause this, my system is pretty fresh right now. Sidenote, the home directory on Silverblue is /var/home/[user]

**Comment by mirkobrombin** [comment]
Is still accessible from /home. 

I don’t have idea but I’ll keep this open, so maybe other users have the same problem.
